core/templates/dev/head/css/oppia.css

```diff
@@ -358,6 +358,17 @@ p:last-child {
   margin: 1.4em 0;
 }
 
+/* This will prevent the enlargment of the text in the RTE that is
+  nested inside the p and span tag. !important is added to override
+  the inline css.*/
+.oppia-rte p span {
+  font-size: 1em !important;
```

Comment:
Is this the only way to get this done? I am saying this because we already have an issue #5097 regarding removal of `!important` from the codebase.

Reply:
I agree with @YashJipkate. Please find a way to avoid !important. Typically you can do this by adding more specificity in the CSS rule.

Reply:
See e.g. https://uxengineer.com/css-specificity-avoid-important-css/#incremental-specificty

Reply:
I don't think we can remove `!important` from here. As discussed on the issue thread, a span tag gets introduced before the text (which the issue of CKEditor) with a inline style. `!important` is a only way to remove that inline style.

Reply:
Wait -- I don't understand something. If the inline style is coming from a CSS class in oppia.css, why is that style being added inline to the span tag? Is there some JS code that is doing that? Normally styles in oppia.css classes will show up as being part of the class in the CSS inspector, it wouldn't be "unpacked" and added to the DOM element directly.

Reply:
The span tag is introduced by CKEditor. It comes as `style=font-size:1.6 em`. It's taking the same font size whatever is mentioned in the body tag in oppia.css.

Reply:
Ah ok. Can you add all this information to the comment as justification of why this cannot be resolved without using !important? Once you've done that, I think this can go in.

Thanks!

Reply:
Sure, thanks!